src/bulls_and_cows_lib/board.cpp

```diff
@@ -4,5 +4,41 @@
 namespace bulls_and_cows {
 
     // TODO: define the body of the functions declared in board.cpp
+    // Create a board and with a new random secret code composed of allowed characters
+    Board create_board(const GameOptions& game_options)
+    {
+        //game_option =
+    }
+
+    // Validate that a user-entered code contains proper number of allowed characters
+    bool validate_attempt(const GameOptions& game_options, const Code& attempt)
+    {
+    }
+
+    // Compare a user-entered code with the secret code and give the corresponding number of bulls and cows
+    Feedback compare_attempt_with_secret_code(const Code& attempt, const Code& secret_code)
+    {
```

Comment:
Here, you must :
- compare the characters of attempt.value and secret_code.value
- A "bull" is a character in attempt.value that is the same as in secret_code.value and at the same location
- A "cow" is a character in attempt.value that is the same as in secret_code.value but at different location
- You must return a feedback containing the number of bulls and cows.